src/desktop/apps/conversations/server.tsx

```diff
@@ -0,0 +1,54 @@
+import { buildServerApp } from "reaction/Artsy/Router/server"
+import { stitch } from "@artsy/stitch"
+import { conversationRoutes } from "reaction/Apps/Conversation/routes"
+import React from "react"
+import { buildServerAppContext } from "desktop/lib/buildServerAppContext"
+import express, { Request, Response, NextFunction } from "express"
+
+export const app = express()
+
+app.get(
+  "/user/conversations",
```

Comment:
@ashkan18 - `/user/conversations/*` is the fix!

Reply:
Since you're not explicitly handling the dyanmic path (`:id`) here in express, but rather deferring it to the routing framework, a `*` will match all following paths.